avalanche/deployments/utils/dependencies.ts

```diff
@@ -87,6 +87,23 @@ export default {
     43114: "",
   },
 
+  // Set Protocol System
+  CONTROLLER: {
+    43114: "0x14099863F0B6490759f9D8bC5653CF52b7eF38eb",
+  },
+  SET_TOKEN_CREATOR: {
+    43114: "0xE99447aBbD5A7730b26D2D16fCcB2086319e4bC3",
+  },
+  DEBT_ISSUANCE_MODULE_V2: {
+    43114: "0x683A3F97Baf396d3111415267D2A02df8503D0e1", // Needs to be replaced
+  },
+  STREAMING_FEE_MODULE: {
+    43114: "0x4F70287526ea9Ba7e799D616ea86635CdAf0de4F",
+  },
+  TRADE_MODULE: {
+    43114: "0x44279c2cbFaa14D354144895B0c7771c597d6944",
+  },
```

Comment:
Sorry overlooked this ... we'll have to track production and staging addresses here like in the Polygon folder:

https://github.com/SetProtocol/set-v2-strategies-deployments/blob/0443dc165111cf3bb29d824ab71e25b953b9b4d1/polygon/deployments/utils/dependencies.ts#L143-L180

Reply:
Lgtm with [d089ff3](https://github.com/SetProtocol/set-v2-strategies-deployments/pull/42/commits/d089ff3564c73f81ff30697b301638a2d35c737f)